app/code/Magento/Catalog/Test/Unit/Model/ResourceModel/Category/AggregateCountTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Catalog\Test\Unit\Model\ResourceModel\Category;
+
+use Magento\Catalog\Model\Category;
+use Magento\Catalog\Model\ResourceModel\Category\AggregateCount;
+use Magento\Catalog\Model\ResourceModel\Category as ResourceCategory;
+use Magento\Framework\DB\Adapter\AdapterInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Aggregate count model test
+ */
+class AggregateCountTest extends TestCase
+{
+
+    /**
+     * @var AggregateCount
+     */
+    protected $aggregateCount;
+
+    /**
+     * @var ObjectManagerHelper
+     */
+    protected $objectManagerHelper;
+
+    /**
+     * @var Category|MockObject
+     */
+    protected $categoryMock;
+
+    /**
+     * @var  ResourceCategory|MockObject
+     */
+    protected $resourceCategoryMock;
+
+    /**
+     * @var AdapterInterface|MockObject
+     */
+    protected $connectionMock;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUp()
```

Comment:
Hello @ProkopovVitaliy,
could you please add a return type definition here for compatibility with PHPUnit 8?
```suggestion
    public function setUp(): void
```